puppet/zulip_ops/files/zulip-ec2-configure-interfaces

```diff
@@ -54,14 +54,17 @@ import sys
 import boto.utils
 import netifaces
 from six.moves import range
+if False: from typing import Optional
```

Comment:
This is failing a PEP-8 lint rule because the interior should be on a second line.